src/color-slider/color-slider.js

```diff
@@ -37,7 +38,7 @@ export default class ColorSlider extends HTMLElement {
 
 			this.#initialized = true;
 
-			this._el.slider.dispatchEvent(new Event("input"));
+			this._el.slider.dispatchEvent(new Event("colorchange"));
```

Comment:
```suggestion
			this._el.slider.dispatchEvent(new ColorChangeEvent("colorchange", {bubbles: true, color: this.color}));
```

Reply:
Based on my experiments, this (partially) fixes the issue.

Reply:
Gah, I knew it must be something silly like that. Thanks Dmitry!